lib/active_shipping/carriers/fedex.rb

```diff
@@ -472,10 +472,15 @@ def parse_rate_response(origin, destination, packages, response, options)
 
       success = response_success?(xml)
       message = response_message(xml)
+      missing_xml_field = false
 
       if success
         rate_estimates = xml.root.css('> RateReplyDetails').map do |rated_shipment|
-          service_code = rated_shipment.at('ServiceType').text
+          service_code = rated_shipment.at('ServiceType').try(:text)
+          unless service_code.present?
+            missing_xml_field = true
+            next
+          end
```

Comment:
@christianblais Indeed.
I've added a rescue/begin and kept the same "return what we can" logic